### ISSUE TITLE
[Feature Request] Participants list download

### ISSUE THREAD

**Issue by ihsaan-ullah** [codalab/codabench#1470]
In one competition, we need to download the participants list for their emails. Do you think this will be useful in general?

**Comment by Didayolo** [comment]
Yes, if this is implemented somewhere, we should merge it to the code. I guess a button `Download as CSV` next to `Email all participants` would do it.

**Comment by georg-brunner** [comment]
I would find that useful too!
